docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '3'
+networks: 
+  chapter:
```

Comment:
Don't necessarily need to specify a `network` like this, [by default Compose sets up a single network](https://docs.docker.com/compose/networking/)